README.md

```diff
@@ -0,0 +1,96 @@
+
+#### 2.2.2. Fill in your backend endpoints
+
+Under `Project settings -> Web component` configure the endpoints as seen in the following image, but use your own ngrok url (the paths stay the same).
+```
+<your-ngrok-url>/api/sessionToken
+<your-ngrok-url>/api/loginInfo
+<your-ngrok-url>/api/passwordVerify
+```
+![image](https://user-images.githubusercontent.com/23581140/205945743-207cd062-bb41-4b3c-af0c-cb13bf279f9c.png)
+
+#### 2.2.3. Authorize origins
+Inside the developer panel under `Project settings -> REST API` you need to enter the following origins in order to allow them to share resources.
+1. `https://CNAME`, which represents the Corbado web component
+2. `http://localhost:8000`, which is where your backend instance is running
+3. Your ngrok url which connects your local backend to the world
+
+![image](https://user-images.githubusercontent.com/23581140/205950485-6285d536-d676-4382-a23c-c3c0bbfe3de4.png)
+
+### 2.3. Setup
+To start the local development server your system requires [PHP](https://www.php.net/manual/en/install.php) and [Symfony](https://symfony.com/download):
+```
+brew install php
+brew install symfony-cli/tap/symfony-cli
+```
+
+You then need to install [Composer](https://getcomposer.org/download/):
+```
+curl -sS https://getcomposer.org/installer | php
+```
+With Composer you should be able to install all required packages for this project:
+```
+sudo -E env "PATH=$PATH" php composer.phar install
+```
+
+You should now be able to run this demo by typing `symfony server:start` into your console while being located in the same folder that contains this README file. If the setup was done correctly the following messages should appear in your terminal:
+![image](https://user-images.githubusercontent.com/23581140/205909459-7ed3d679-b313-40d3-85be-1178b80a1594.png)
+To verify that your instance is running without errors enter `http://localhost:8000/ping` into your browser. If "pong" is displayed you can continue with the next step.
+
+
+The endpoints of your local system have to be public so Corbado can send requests there. To make your local instance publicly availbale we use [ngrok](https://ngrok.com/download). It can be installed using:
+```
+curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+```
+
+You can start your ngrok instance by typing `ngrok http 8000`. In your terminal you should see the following:
+![image](https://user-images.githubusercontent.com/23581140/205919914-986f95ea-7c32-4501-a651-f47b16e3b2e2.png)
+
+Entering the url which is inside the red rectangle with `/ping` as path should now display "pong" as well since this ngrok url just forwards requests to your local instance.
+
+A few last steps:
```

Comment:
Wieso switchen wir jetzt hier vom Format in so kleine Aufzählungen und bleiben nicht bei unserer Nummerierung -> das muss konsistent sein!